webapp/src/components/post_type_zoom/post_type_zoom.jsx

```diff
@@ -102,7 +102,7 @@ export default class PostTypeZoom extends React.PureComponent {
             if (props.meeting_personal) {
                 subtitle = (
                     <span>
-                        {'Personal Meeting ID (PMI): '}
+                        {'Personal Meeting ID (PMI) : '}
```

Comment:
This change probably can stay, depending on how clean you want to make the revert..